docs/index.md

```diff
@@ -9,10 +9,11 @@ description: Besu is an open-source Ethereum client developed under the Apache 2
 
 ## What is Hyperledger Besu?
 
-Hyperledger Besu is an open-source Ethereum client developed under the Apache 2.0 license and
-written in Java. It runs on the Ethereum public network, private networks, and test networks such
-as Rinkeby, Ropsten, and Görli. Besu implements proof of work (Ethash) and proof of authority (IBFT
-2.0, Clique, and QBFT) consensus mechanisms.
+Hyperledger Besu is an open-source Ethereum client developed under the Apache 2.0 license and written in Java.
+It runs on Ethereum Mainnet, private networks, and test networks such as Rinkeby, Ropsten, Goerli, and Kiln.
+Besu serves as an [execution client](Concepts/Merge.md) on Ethereum Mainnet and the Kiln testnet.
```

Comment:
The Merge testnet? Not sure if any additional testnets will be created, but it's a possibility I suppose.